### PR TITLE
Fix escape warnings.

### LINKED ISSUE #1906: Remove regex deprecation warnings
When running `make tests` on master (8efa5ad190) I see those warnings: (not much information about line)

@glasserc I remember you found some in #1758 
``` 
<unknown>:1
  <unknown>:1: DeprecationWarning: invalid escape sequence \]
  <unknown>:1: DeprecationWarning: invalid escape sequence \]
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \]
  <unknown>:1: DeprecationWarning: invalid escape sequence \]
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \.
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \?
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[
  <unknown>:1: DeprecationWarning: invalid escape sequence \[

```

### PR DESCRIPTION
**Pull request by Natim**
Fixes #1906 

- [ ] Add a changelog entry.

### CODE REVIEW

File: kinto/core/resource/__init__.py [hunk 1]
```diff
@@ -1028,7 +1028,7 @@ def _extract_sorting(self, limit):
         modified_field_used = self.model.modified_field in specified
         for field in specified:
             field = field.strip()
-            m = re.match(r"^([\-+]?)([\w\.]+)$", field)
+            m = re.match(r"^([\\-+]?)([\\w\\.]+)$", field)
```
Comment (Natim) [inline kinto/core/resource/__init__.py]: Maybe changing from `"` to `'` would be a better fix?

Comment (leplatrem) [reply kinto/core/resource/__init__.py]: Warnings are gone but tests fail :)

Comment (leplatrem) [reply kinto/core/resource/__init__.py]: I don't know what we have in other regexp... single quotes are fine IMO

**Comment by Natim** [comment]
Can you check that it fixed it?

**Comment by leplatrem** [comment]
Yes it did:
https://travis-ci.org/Kinto/kinto/jobs/461791830#L606

The tests output a loooot of warnings (https, postgresql config...)

**Comment by Natim** [comment]
For a one char commit? :joy:

**Pull request merged**